### ISSUE TITLE
iPad landscape rotation incorrect

### ISSUE THREAD

**Issue by RickPasveer** [AlexLittlejohn/ALCameraViewController#129]
When you open the camera view on iPad in landscape mode the camera view rotation is incorrect.

after you rotate the iPad the camera view is correct again.

**Comment by KanybekMomukeyev** [comment]
@RickPasveer same issue on ipad. did you find any solution?

**Comment by KanybekMomukeyev** [comment]
just added  self.rotateCameraView() in  viewDidAppear
```
public override func viewDidAppear(_ animated: Bool) {
        super.viewDidAppear(animated)
        self.rotateCameraView()
        if cameraView.session?.isRunning == true {
            notifyCameraReady()
        }
    }
```

**Comment by MOLO17** [comment]
same problem here, the camera rotate but the capture output is in a wrong rotation state

**Comment by jaceee** [comment]
Any updates on this?

**Comment by GreaseMonk** [comment]
To reproduce, start making a picture while the iPad is in landscape mode.
The camera view is rotated as in when you tilt the iPad forward and backward, the screen will go left/right.

+1

**Comment by WillemPieterse** [comment]
Same problem here... 
Some (pensioner) users of my App cannot remember the work-around of going back, rotating to portrait and activating the camera again...

Kindly fix. I can see some heart-attacks on the (landscape) horizon

**Comment by GreaseMonk** [comment]
Any ETA on a fix for this ?

**Comment by dneykov** [comment]
Any updates on this?

**Comment by AlexLittlejohn** [comment]
I will look into it but I have to buy an iPad first 😃

**Comment by dneykov** [comment]
@AlexLittlejohn adding ```self.rotateCameraView()``` in viewDidAppear does the trick. 
Check @KanybekMomukeyev answer above.

Actually the problem is just in camera view. When I capture the image, the image is in correct orientation

**Comment by GreaseMonk** [comment]
Thank you for looking into it @AlexLittlejohn can you notify here when we can update the podfile with the bugfix and which version it is ?
💪 💪 💪 💪 💪 💪 💪

**Comment by AlexLittlejohn** [comment]
Push a new release v2.0.1 to resolve this issue

**Comment by GreaseMonk** [comment]
@AlexLittlejohn tested in 2.0.1
result remains the same.
Download & watch video: https://drive.google.com/file/d/0B7kvAdrSP6XkcmtvUlZ4d0lJUnM

this behaviour is the same on iPad Air 2.

**Comment by GreaseMonk** [comment]
Code supplement

`@IBAction func takePicture(_ sender: AnyObject) {
        let croppingParameters = CroppingParameters(isEnabled: true, allowResizing: false, allowMoving: true)
        let cameraViewController = CameraViewController.init(croppingParameters: croppingParameters, allowsLibraryAccess: true, allowsSwapCameraOrientation: true, allowVolumeButtonCapture: true, completion: { [weak self] image, asset in
            if let img = image {
                self?.chosenPhoto = img
            }
            
            self?.tableView.reloadData()
            
            self?.dismiss(animated: true, completion: nil)
        })
        present(cameraViewController, animated: true, completion: nil)
    }`

**Comment by AlexLittlejohn** [comment]
I've fixed this issue in 2.0.2. Tested on an actual iPad
